xtask/src/builder.rs

```diff
@@ -323,7 +337,10 @@ impl Builder {
         }
         self
     }
-
+    ///// Add an app binary build out of souce tree
```

Comment:
I think this was probably just a placeholder for a method that was never used? if so go ahead and remove it from the patch.